### PR TITLE
Fix Gmail and Apple Mail Mobile Repsonsiveness (Header and Greetings)

### PR DESCRIPTION
**Pull request by eliasfang**
### Administrative Info
Monday Board ID: 940408300
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Switched header into a banner image (instead of lightbulbs and text)
- Fixed padding of header
- Changed width of greetings text to 100% to ensure that it is centered in mobile email clients
- Altered widths of socials section to fit mobile email clients

### Testing
How did you confirm your changes worked? 
- Sent test emails and checked on desktop and mobile clients
- Gmail still inverted on dark mode :((

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Desktop Gmail | Mobile Gmail | Mobile Apple |
| --- | --- | --- |
| image | IMG_9175 | IMG_9174 |

### CODE REVIEW

File: py/head.py [hunk 1]
```diff
@@ -21,15 +21,9 @@ def _head():
...
-                img(cls='lightbulb', src='https://drive.google.com/uc?export=view&id=1gq2ARk8HOCKw9MBrUfrEYe3GJjHGyBta')
-            # TSE
-            with table(cls='column columnTitle').add(tr()).add(td(cls='padding')).add(table()).add(tr()).add(td()):
-                h1('TRITON SOFTWARE ENGINEERING')
-            # Right lightbulb
-            with table(cls='column columnLogo').add(tr()).add(td(cls='padding')).add(table()).add(tr()).add(td()):
-                img(cls='lightbulb', src='https://drive.google.com/uc?export=view&id=1gq2ARk8HOCKw9MBrUfrEYe3GJjHGyBta')
+            # TSE Banner
+            with table().add(tr()).add(td(cls='padding')).add(table()).add(tr()).add(td(cls='center')):
+                img(cls='banner', src='https://drive.google.com/uc?export=view&id=1czSvLEq4b05e2ngXvCwL5OUUz-G92X_J')
```
Comment (wes-chen) [inline py/head.py:26]: Will this link be constant forever, or should we putting the link as a var declared at the top of this file so it's easy to change if the TSE banner needs editing?

File: py/newsletter.py [hunk 2]
```diff
@@ -29,15 +29,15 @@
...
-            _foot()
+            _greetings(greet_title)
+        # with tr().add(td()):
+        #     _bodylineleft()
+        # with tr().add(td()):
+        #     _bodylineright()
+        # with tr().add(td()):
+        #     _bodynoline()
+        # with tr().add(td()):
+        #     _foot()
```
Comment (wes-chen) [inline py/newsletter.py:40]: remove all of these comments

**Review by wes-chen — CHANGES_REQUESTED** [review]

**Review by shravan1k — CHANGES_REQUESTED** [review]
Two recommended changes I have are adding an alt to the image and removing the image from the folder as it is already hosted on google drive. Otherwise, looks good!

File: py/head.py (hunk 1 above)
Comment (eliasfang) [reply py/head.py:26]: Done

[the remark this answers is not in the document]
Comment (eliasfang) [reply py/head.py:26]: Good idea. Fixed!

File: py/newsletter.py (hunk 2 above)
Comment (eliasfang) [reply py/newsletter.py:40]: Removed!

**Review by shravan1k — APPROVED** [review]
LGTM

File: newsletter.css [hunk 3]
```diff
@@ -202,6 +202,7 @@ a {
     text-align: -webkit-center;
 }
 
+
```
Comment (wes-chen) [inline newsletter.css:205]: what is this?

File: newsletter.css [hunk 4]
```diff
@@ -279,23 +256,23 @@ a {
     font-weight: normal;
     font-size: 16px;
     letter-spacing: 0.02em;
-    text-align: center;
 
-    margin-left: 10px;
+    margin-left: 7px;
+
```
Comment (wes-chen) [inline newsletter.css:261]: not that I'm looking _super_ hard at css files, but there's a lot of weird whitespaces everywhere

**Review by wes-chen — APPROVED** [review]
small nits, but LGTM. no need to request review again after you delete the weird newlines everywhere

File: newsletter.css (hunk 4 above)
Comment (eliasfang) [reply newsletter.css:261]: Ah yes I thought the whitespaces helped separate the sections but technically that's redundant since I have comments. Thanks!

**Pull request merged**